client/src/pages/reports/Compact.js

```diff
@@ -0,0 +1,1012 @@
+  left: 0mm;
+  display: flex;
+  width: 100%;
+  max-height: 50px;
+  margin: 10px auto;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1rem;
+  -webkit-print-color-adjust: exact !important;
+  color-adjust: exact !important;
+  img {
+    max-width: 50px !important;
+    max-height: 24px !important;
+  }
+  & * {
+    color: white !important;
+  }
+  @media print {
+    position: fixed;
+    max-height: 70px;
+  }
+`
+
+const HeaderContent = styled.div`
+  ${HF_COMMON_STYLE};
+  top: 0mm;
+  border-bottom: 1px solid black;
+  background-color: ${props => props.bgc} !important;
+`
+
+const FooterContent = styled.div`
+  ${HF_COMMON_STYLE};
+  bottom: 0mm;
+  border-top: 1px solid black;
+  background-color: ${props => props.bgc} !important;
+`
+
+const PrintedByBox = styled.span`
+  align-self: flex-start;
+  width: auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-items: flex-end;
+  flex-wrap: wrap;
+  font-size: 10px;
+  & > span {
+    display: inline-block;
+    text-align: right;
+  }
+`
+
+const ClassificationBanner = () => {
+  return (
+    <ClassificationBannerBox>
+      <CompactSecurityBanner />
+    </ClassificationBannerBox>
+  )
+}
+
+const ClassificationBannerBox = styled.div`
+  width: auto;
+  max-width: 67%;
+  text-align: center;
+  display: inline-block;
+  & > .banner {
+    padding: 2px 4px;
+  }
+`
+const CompactTable = ({ children }) => {
+  return (
+    <CompactStyledTable>
+      <thead>
+        <tr>
+          <SpacedTd colSpan="2" />
+        </tr>
+      </thead>
+      <tbody>{children}</tbody>
+      <tfoot>
+        <tr>
+          <SpacedTd colSpan="2" />
+        </tr>
+      </tfoot>
+    </CompactStyledTable>
+  )
+}
+
+CompactTable.propTypes = {
+  children: PropTypes.node
+}
+
+const CompactStyledTable = styled.table`
+  width: 100%;
+`
+
+const SpacedTd = styled.td`
+  height: 80px;
+`
+
+export const ROW_TYPES = {
+  titleLike: "titleLike",
+  onlyData: "onlyData"
+}
+
+export const CompactRow = ({ label, content, rowType, ...otherProps }) => {
+  const { style, className } = otherProps
+  const CustomStyled = styled(ROW_STYLE)`
+    ${style};
+  `
+  if (rowType === ROW_TYPES.titleLike) {
+    return (
+      <CustomStyled>
+        <RowLabel colSpan="2">{label}</RowLabel>
+      </CustomStyled>
+    )
+  }
+
+  if (rowType === ROW_TYPES.onlyData) {
+    return (
+      <CustomStyled>
+        <RowContent colSpan="2">{content}</RowContent>
+      </CustomStyled>
+    )
+  }
+
+  const lowerLabel =
+    typeof label === "string" ? label.toLocaleLowerCase() : label
+  const topHeader = className === "reportField"
+
+  return (
+    <CustomStyled className={className || null}>
+      <RowLabel topHeader={topHeader}>{lowerLabel}</RowLabel>
+      <RowContent>{content}</RowContent>
+    </CustomStyled>
+  )
+}
+
+CompactRow.propTypes = {
+  label: PropTypes.node,
+  content: PropTypes.node,
+  rowType: PropTypes.string
```

Comment:
Can be more specific such as `rowType: PropTypes.oneOf(Object.values(ROW_TYPES))`

Reply:
I have logic where if none of the ROW_TYPES supplied, we go with general row styling. So if i want to go with normal row styling, i pass nothing, which would fail here